lecli/saved_query_api.py

```diff
@@ -0,0 +1,191 @@
+"""
+Saved Query API module.
+"""
+import click
+import requests
+
+from lecli import api_utils
+from lecli import response_utils
+
+
+def _url():
+    """
+    Get rest query url of account resource id.
+    """
+    return 'https://rest.logentries.com/query/saved_queries'
+
+
+def _pretty_print_saved_query(query):
+    """
+    Pretty print saved query object
+    :param query:
+    """
+    click.echo("Name: \t%s" % query['name'])
+    click.echo("Logs: \t%s" % ",".join(query['logs']))
+    click.echo("ID: \t%s" % query['id'])
+    click.echo("LEQL \tStatement: \t%s" % query['leql']['statement'])
+    click.echo("\tTime range: \t%s" % query['leql']['during'].get('time_range'))
+    click.echo("\tFrom: \t\t%s" % query['leql']['during'].get('from'))
+    click.echo("\tTo: \t\t%s" % query['leql']['during'].get('to'))
+    click.echo("**********************************************")
+
+
+def _pretty_print_saved_query_error(response):
+    """
+    Pretty print saved query error
+    """
+    try:
+        error_body = response.json()
+        if 'fields' in error_body:
+            click.echo('Invalid field: %s' % ",".join(response.json()['fields']))
+        if 'messages' in error_body:
+            click.echo('Message: %s' % ",".join(response.json()['messages']))
+    except ValueError:
+        exit(1)
+
+
+def _handle_saved_query_response(response):
+    """
+    Handle saved query response, check whether there are multiple entities in the response or not
+    :param response:
+    """
+    if response.json().get('saved_queries'):
+        queries = response.json()['saved_queries']
+        # print tabulate(queries, headers={})
+        for query in queries:
+            _pretty_print_saved_query(query)
+    elif response.json().get('saved_query'):
+        query = response.json()['saved_query']
+        _pretty_print_saved_query(query)
+
+
+def get_saved_query(query_id=None):
+    """
+    Get a specific saved query
```

Comment:
Yeah that's true, we had two functions before I created the PR but I wanted to dry it up because of code duplication apart from the url part. I will change the comment.